src/main/java/core/basesyntax/service/impl/ShopService.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import java.util.Map;
+
+public interface ShopService {
```

Comment:
Interfaces should be placed in the `service` package, not `impl`. The `impl` package is typically used for concrete implementations of the interfaces.